### DISCUSSION TITLE
NET MAUI

### DISCUSSION THREAD

**Discussion by theerudito** [dotnet/maui#17526] [General]
how i get the path 
 on NET MAUI Android.OS.Environment.GetExternalStoragePublicDirectory(Android.OS.Environment.DirectoryDocuments).AbsolutePath;

**Comment by acaliaro** [comment]
You can use it directly in your ViemModel with #ifdef ANDROID or creating an interface and implement in platform specific directories

**Reply by theerudito** [reply]
maybe some example doesn't work
